doc/man3/EVP_blake2b512.pod

```diff
@@ -0,0 +1,64 @@
+=pod
+
+=head1 NAME
+
+EVP_blake2b512,
+EVP_blake2s256
+- BLAKE2 For EVP
+
+=head1 SYNOPSIS
+
+ #include <openssl/evp.h>
+
+ const EVP_MD *EVP_blake2b512(void);
+ const EVP_MD *EVP_blake2s256(void);
+
+=head1 DESCRIPTION
+
+BLAKE2 is an improved version of BLAKE, which was submitted to the NIST SHA-3
+algorithm competition.
+
+=over 4
+
+=item EVP_blake2s256()
+
+The BLAKE2s algorithm that produces a 256-bit output from a given input.
+
+=item EVP_blake2b512()
+
+The BLAKE2b algorithm that produces a 512-bit output from a given input.
+
+=back
+
+=head1 RETURN VALUES
+
+These functions return a B<EVP_MD> structure that contains the
+implementation of the symmetric cipher. See L<EVP_MD_meth_new(3)> for
+details of the B<EVP_MD> structure.
+
+=head1 CONFORMING TO
+
+N/A.
```

Comment:
In a case like this, I think it would be preferable to remove this section.  What's your motivation for keeping it?

Reply:
Just for consistency among all the files. We can remove it alright if it is preferred.

Reply:
It is at least my preference.  I don't think we've ever had a N/A section before, so this is a new situation for us.  What says the rest of @openssl/omc and @openssl/committers ?

Reply:
How about referring to https://tools.ietf.org/html/rfc7693 ?

Reply:
Good call @bernd-edlinger , thank you! Maybe the answer to @levitte is : there can always be a reference...

Reply:
Resolved.

Reply:
FWIW I write man pages in my other life as a FreeBSD documentation committer, and CONFORMING TO is not a required section, so it should be omitted when empty.

Reply:
yes, but it would normally be the case that it conforms to something, like an RFC or something.